src/kOS/Module/kOSProcessor.cs

```diff
@@ -628,12 +637,29 @@ private void ProcessElectricity(Part partObj, float time)
         {
             if (ProcessorMode == ProcessorModes.OFF) return;
 
-            RequiredPower = shared.VolumeMgr.CurrentRequiredPower;
-            var electricReq = time * RequiredPower;
-            var result = partObj.RequestResource("ElectricCharge", electricReq) / electricReq;
-
-            var newMode = (result < 0.5f) ? ProcessorModes.STARVED : ProcessorModes.READY;
-            SetMode(newMode);
+            double volumePower = 0;
+            var volume = shared.VolumeMgr.CurrentVolume;
+            if (volume.Name == "Archive")
+            {
+                // TODO: Like with the default value of ECPerBytePerSecond, this corresponds to the existing value.
+                // It should be adjusted for balance.
+                volumePower = 50000 * ECPerBytePerSecond;
```

Comment:
Magic number here.  that 50k should have a name, so it's immediately clear what it represents.